named-addresses/mainnet_sapphire.json

```diff
@@ -0,0 +1,187 @@
+    },
+    {
+        "Name": "Pending Withdrawal Account",
+        "Address": "oasis1qr677rv0dcnh7ys4yanlynysvnjtk9gnsyhvm6ln",
+        "Description": "Withdrawal from the ParaTime is performed in two steps. This account holds the tokens that are still waiting for the consensus transaction to be confirmed."
+    },
+    {
+        "Name": "Pending Delegation Account",
+        "Address": "oasis1qzcdegtf7aunxr5n5pw7n5xs3u7cmzlz9gwmq49r",
+        "Description": "Delegation from the ParaTime is performed in two steps. This account holds the tokens that are still waiting for the consensus transaction to be confirmed."
+    },
+    {
+        "Name": "Zero Address",
+        "Address": "oasis1qq2v39p9fqk997vk6742axrzqyu9v2ncyuqt8uek",
+        "Description": "This address is the oasis-standard derivation/counterpart of the Ethereum address 0x0. Sometimes used in internal data representations to indicate the absence of an account. For example, token mints are encoded as transfers from this address, and token burns as transfers to this address."
+    },
+    {
+        "Name": "Subcall Precompile",
+        "Address": "oasis1qzr543mmela3xwflqtz3e0l8jzp8tupf3v59r6qn",
+        "Description": "Interacts with Oasis Runtime SDK modules. Some transactions are performed in two steps and emit event in the next block."
+    },
+    {
+        "Name": "Rosy",
+        "Address": "oasis1qrysq46guu3lug3mge6vgg7mrtthfuqnucq7rnp0",
+        "Description": "ERC20"
+    },
+    {
+        "Name": "illumineX Token",
+        "Address": "oasis1qzhxwvks20f6fng566rzcs98t65qjnp4pgd5mnvl",
+        "Description": "ERC20"
+    },
+    {
+        "Name": "RA1L",
+        "Address": "oasis1qzvys6j3vpe8sq7az60z3p24xtcxdxyaccjndpdt",
+        "Description": "ERC721"
+    },
+    {
+        "Name": "NFTrout",
+        "Address": "oasis1qpwe6h0hw6s0x4skujhznld065300p83wgxnvu0n",
+        "Description": "ERC721"
+    },
+    {
+        "Name": "Rosalie",
+        "Address": "oasis1qz7uqg8979ucy9c2l9vqudvcnt3p08th4ua7fqhl",
+        "Description": "ERC20"
+    },
+    {
+        "Name": "Ocean Token",
+        "Address": "oasis1qr43dvghx4a75vndjcsg2r9ct2xqftwrgcnt5tge",
+        "Description": "ERC20"
+    },
+    {
+        "Name": "confiTheDog.com",
+        "Address": "oasis1qzdm0aqruaarc0lmdt802sha5sv97um5rcdn8gh5",
+        "Description": "ERC20"
+    },
+    {
+        "Name": "Wrapped ROSE",
+        "Address": "oasis1qpdgv5nv2dhxp4q897cgag6kgnm9qs0dccwnckuu",
+        "Description": "ERC20"
+    },
+    {
+        "Name": "PERRY",
+        "Address": "oasis1qqu9s8uvk7rawj0ugmt7hq3vhehcerdswvjjt9hp",
+        "Description": "ERC20"
+    },
+    {
+        "Name": "illuminex pair",
+        "Address": "oasis1qpaeymv9ly8gqweg2qvpnd2ufcj33vgdyqqlruqw",
+        "Description": "ERC20"
+    },
+    {
+        "Name": "wstROSE",
+        "Address": "oasis1qr2ta7ussnps40fnmp58p04cwn7yvr56eqepa7xh",
+        "Description": "ERC20"
+    },
+    {
+        "Name": "Sapphire Inu",
```

Comment:
Same question here; I thought we would just have a couple of known/trusted addresses here, not all contracts. 
Some of the token names are quite apparently non-serious/spam and don't belong in any "official list" like this one.

Reply:
agree, lists need more removals. Will try to find someone who can decide which tokens are "trusted" for us.

Reply:
based on LJU Summit need to get feedback from Ed first

Reply:
https://github.com/oasisprotocol/nexus/pull/798/commits/d4d5513766923040167ba290e039d04ca2fae25f commit has a list filtered by Ed